packages/dbos-cloud/applications/deploy-app-code.ts

```diff
@@ -240,6 +240,9 @@ export async function deployAppCode(
     if (isCloudAPIErrorResponse(axiosError.response?.data)) {
       handleAPIErrors(errorLabel, axiosError);
       const resp: CloudAPIErrorResponse = axiosError.response?.data;
+      if (resp.DetailedError) {
+        logger.error(resp.DetailedError)
+      }
```

Comment:
There's a case to be made for making the output less verbose on demand. We'll see.